.gitignore

```diff
@@ -19,3 +19,4 @@ build/_output/
 coverage*.out
 report_e2e.json
 gosec.json
+.vscode/
```

Comment:
This should go in the same place as in `stolostron` to prevent merge conflicts in the future.